lib/default.nix

```
@@ -23,7 +23,7 @@ let
   # replaces `.pw.toml` extensions with `.jar`
   # to correct the store paths of jarfiles
   # string -> string
-  fixupName = replaceStrings [".pw.toml"] [".jar"];
+  withJarExtension = name: replaceStrings [".pw.toml"] [".jar"] name;
```

```suggestion
  withJarExtension = name: pkgs.lib.strings.sanitizeDerivationName (replaceStrings [".pw.toml"] [".jar"] name);
```
I think we could probably sanitize here directly as opposed to doing it for almost every call. Only real effect it would have on end users is differing file paths with the `nix-minecraft` module, but that should be fine